pytition/petition/views.py

```diff
@@ -1605,6 +1605,9 @@ def transfer_petition(request, petition_id):
             except:
                 notFound = True
         if owner_type == "user":
+            if settings.DISABLE_USER_PETITION:
+                messages.error(request, _("Users are not allowed to be transfered a petition."))
```

Comment:
I would say "Users are not allowed to transfer petitions to organizations on this instance"

Reply:
```suggestion
                messages.error(request, _("Users are not allowed to transfer petitions to organizations on this instance."))
```